config/docker/local-docker-compose.yml

```diff
@@ -2,27 +2,26 @@ version: '3.4'
 
 services:
   selenium_hub:
-    image: 'selenium/hub:3.11.0-californium'
+    image: selenium/hub:3.14.0-helium
     environment:
       GRID_THROW_ON_CAPABILITY_NOT_PRESENT: 'false'
       GRID_MAX_SESSION: 10
       GRID_BROWSER_TIMEOUT: 100000
       GRID_TIMEOUT: 90000
-      GRID_NEW_SESSION_WAIT_TIMEOUT: 300000
+      GRID_NEW_SESSION_WAIT_TIMEOUT: 150000
     ports:
       - '4444:4444'
+
   chrome:
-    image: 'selenium/node-chrome:3.11.0-californium'
+    image: selenium/node-chrome:3.14.0-helium
     volumes:
       - /dev/shm:/dev/shm
-    ports:
-      - '5555:5555'
     deploy:
-      replicas: 5
+      replicas: 1
```

Comment:
if replicas are set to 1 can we remove the deploy settings all together? (i assume 1 is the default)